cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -227,6 +229,10 @@ type ControllerContext struct {
 	// InformerFactory gives access to informers for the controller.
 	InformerFactory informers.SharedInformerFactory
 
+	// Extend InformerFactory gives access to informers for the controller w
+	// eg: With {IncludeUninitialized: true} to see all unInitialized resour
+	ExtendInformerFactory informers.SharedInformerFactory
```

Comment:
If it's for uninitialized resources, call it "IncludeUninitializedInformerFactory". "Extend" doesn't help the reader.